config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -249,6 +249,39 @@ presubmits:
       testgrid-tab-name: pr-secrets-store-csi-driver-e2e-gcp
       description: "Run e2e test with gcp provider for Secrets Store CSI driver."
       testgrid-num-columns-recent: '30'
+  - name: pull-secrets-store-csi-driver-e2e-aws
+    decorate: true
+    decoration_config:
+      timeout: 25m
```

Comment:
@lasred Is `25m` sufficient for cluster create + test + delete?

Reply:
No the create itself could take 25 minutes. I update it to 45 minutes